site/snippets/builder/breakgroundSection.php

```diff
@@ -0,0 +1,7 @@
+<section>
```

Comment:
You can delete this file since we are using structured data instead of the builder.